probe-rs-tools/src/bin/probe-rs/main.rs

```diff
@@ -1,3 +1,6 @@
+#![warn(clippy::unnecessary_self_imports)]
```

Comment:
Can we remove those of make them global? :)

Reply:
We can have workspace-global lints but those will apply to everything. This is a bother with `missing_docs` which we really only want on libs.